wd_helper.au3

```diff
@@ -788,7 +788,7 @@ EndFunc   ;==>_WD_LoadWait
 ; ===============================================================================================================================
 Func _WD_Screenshot($sSession, $sElement = Default, $nOutputType = Default)
 	Local Const $sFuncName = "_WD_Screenshot"
-	Local $sResponse, $sResult, $iErr
+	Local $sResponse, $sResult = "", $iErr
```

Comment:
AFAICS, this change isn't needed because this is already accomplished in the `Else` portion of the 2nd If..Else..EndIf.

I've seen you make similar changes before. Would you agree that both coding styles are valid?

Reply:
reverting/fixing